lib/core/src/animation/physics.rs

```diff
@@ -32,13 +32,7 @@ impl DragProperties {
 
```

Comment:
the name `physics.rs` is too general for what is implemented here. I do not have a better name now, but it should be named differently. @farmaazon @mwu-tow @notdanilo this is just a physics engine for objects having mass with attached ruber band and air drag. Any ideas for the name?

Reply:
I don't see why we should rename it. I think it clearly indicates it's a file containing physics based animation components.

Reply:
Oh, sorry for not explaining that clear enough! I mean, when I see `animation/physics` I'm thinking more about a full-blown physics engine, like PhysicsX. This physics engine is very limited one and is able to do much more than just drag object by rubber band attached to it. I believe that a name should not be misleading and I feel that just `physics.rs` gives a first impression of something much more complex than it is. Does it make sense?

Reply:
For me it's clear, as it's part of the animation path, that it is a physics based animation. I wouldn't expect anything too robust in it.

But I am open to change it if you suggest a more suitable name for it.